debian/tests/control

```diff
@@ -0,0 +1,3 @@
+Test-Command: go test -v -mod=vendor ./...
```

Comment:
you used a lot of env variable in debian/rules to pass the tests, those are not needed here?